src/conversions/cart.cpp

```diff
@@ -105,12 +105,15 @@ PROJ_HEAD(cart, "Geodetic/cartesian conversions");
 /*********************************************************************/
 static double normal_radius_of_curvature(double a, double es, double sinphi) {
     /*********************************************************************/
-    if (es == 0)
-        return a;
+    /*Faster without this optimisation** if (es == 0)
+        return a;*/
     /* This is from WP.  HM formula 2-149 gives an a,b version */
     return a / sqrt(1 - es * sinphi * sinphi);
 }
 
+inline double invhypot(const double x, const double y) {
```

Comment:
`invhypot` is used once only, so I do not think it is worth separating out, hence littering the namespace with a function name that is "not entirely obvious"